blocks/cards/cards.js

```diff
@@ -45,6 +53,50 @@ async function localizedTopic(topic) {
   return topicResponse[getLocale()] || topicResponse['en-US'] || topicResponse.identifier || topic;
 }
 
+function closestH3(card) {
```

Comment:
Duplicate?

Reply:
it's not exactly duplicate. there is a slight difference between the `cards.js` and  the sidebar `blog-topics.js` and `blog-years.js`. the function is not super generic and the closest h3 cand depend on the content structure, so that's why I chose to duplicate it between cards and the rest, but reuse the ones from the sidebar between themselves.